compiler/rustc_attr/src/builtin.rs

```diff
@@ -230,11 +231,11 @@ impl UnstableReason {
 /// Returns `None` if no stability attributes are found.
 pub fn find_stability(
     sess: &Session,
-    attrs: &[Attribute],
+    attrs: &SortedIndexMultiMap<u32, Symbol, &Attribute>,
```

Comment:
This type appears in quite a lot of place. It may deserve a type alias.

Reply:
I was also thinking it may deserve a wrapper with some convenience methods, as some patterns are quite common.